### PR TITLE
Automate Settings screen

### PR DESCRIPTION
**Pull request by bilalawan321**
[LEARNER_8180](https://openedx.atlassian.net/browse/LEARNER-8180)

Followings features/stories will be implemented:
Settings screen should show these Header contents:
Back icon
"Setting" heading
Setting Screen should show the following toggle options
Wi-Fi only download
On/OFF toggle
Clicking toggle should show Dialog of "Allow cellular download"
Verify that Dialog should have the following contents:
    Dialog title
    Dialog message
    Allow button
    Dont Allow button
Verify that Clicking Dont Allow button should show Toggel ON
Verify that the Clicking Allow button should show Toggle OFF

### CODE REVIEW

**Comment by bilalawan321** [comment]
@jawad-khan can you please review this PR?

File: tests/android/pages/android_settings.py [hunk 1]
```diff
@@ -0,0 +1,180 @@
+# coding=utf-8
```
Comment (jawad-khan) [inline tests/android/pages/android_settings.py:1]: No need for this line.

File: tests/android/tests/test_android_settings.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                Settings screen should show these Header contents:
+                Back icon
+                "Setting" heading
+                Setting Screen should show the following toggle options
+                Wi-Fi only download
+                On/OFF toggle
+        """
+
+        global_contents = Globals(setup_logging)
+        android_accounts_screen = AndroidAccunts(set_capabilities, setup_logging)
```
Comment (jawad-khan) [inline tests/android/tests/test_android_settings.py:52]: Are we doing any other testing than android here?
if not then no need to prefix every variable with android.

**Review by jawad-khan — APPROVED** [review]

File: tests/android/tests/test_android_settings.py (hunk 2 above)
Comment (bilalawan321) [reply tests/android/tests/test_android_settings.py]: Fixed

File: tests/android/pages/android_settings.py (hunk 1 above)
Comment (bilalawan321) [reply tests/android/pages/android_settings.py]: fixed

**Pull request merged**